src/main/java/com/bfrisco/benchants/enchants/Durability.java

```diff
@@ -25,7 +27,16 @@ public Durability() {
     @EventHandler
     @SuppressWarnings("unused")
     public void onPlayerItemDamageEvent(PlayerItemDamageEvent event) {
-        if (!hasDurability(event.getItem())) return;
+
+        Player player = event.getPlayer();
+        ItemStack item = event.getPlayer().getInventory().getItemInMainHand();
+//*****************Proposed new check, removes nbtapi dependency*********************************
+        if (!ItemInfo.isTitanTool(item)) return;
+        if (!ItemInfo.isActive(item)) return;
```

Comment:
Hmmmm... I don't think a titan tool should have to be toggled on in order to not get durability loss.
That one is up-to you though.